packages/desktop-client/src/components/manager/ManagementApp.js

```diff
@@ -49,170 +49,163 @@ function Version() {
   );
 }
 
-class ManagementApp extends React.Component {
-  constructor(props) {
-    super(props);
-    this.mounted = true;
-    this.history = createBrowserHistory();
-    window.__history = this.history;
-  }
-
-  async componentDidMount() {
+function ManagementApp({
+  files,
+  userData,
+  loadingText,
+  managerHasInitialized,
+  setAppState,
+  getUserData,
+  loadAllFiles,
+}) {
+  const history = createBrowserHistory();
+  window.__history = history;
+
+  // runs on mount only
+  useEffect(() => {
     // An action may have been triggered from outside, and we don't
     // want to override its loading message so we only show the
     // initial loader if there isn't already a message
-    let alreadyLoading = this.props.loadingText != null;
+    let alreadyLoading = loadingText != null;
 
     // Remember: this component is remounted every time the user
     // closes a budget. That's why we keep `managerHasInitialized` in
     // redux so that it persists across renders. This will show the
     // loading spinner on first run, but never again since we'll have
     // a cached list of files and can show them
-    if (!this.props.managerHasInitialized) {
+    if (!managerHasInitialized) {
       if (!alreadyLoading) {
-        this.props.setAppState({ loadingText: '' });
+        setAppState({ loadingText: '' });
       }
-    } else {
-      // If it's not the first time rendering, immediately show the
-      // modal since we should have cached data
-      this.showModal();
```

Comment:
Why was this removed?

Reply:
I forgot to remove it when I deleted the method definition: https://github.com/actualbudget/actual/pull/762/files#diff-5f79220dc9789de60d3ab3e513868b55f7450825ee20364de7bb6059f54fa539L95